website/docs/r/transfer_server.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "aws"
+page_title: "AWS: aws_transfer_server"
+sidebar_current: "docs-aws-resource-transfer-server"
+description: |-
+  Provides a AWS Transfer Server resource.
+---
+
+# aws_transfer_server
+
+Provides a AWS Transfer Server resource.
+
+
+```hcl
+resource "aws_iam_role" "foo" {
+	name = "tf-test-transfer-server-iam-role"
+  
+	assume_role_policy = <<EOF
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+		"Effect": "Allow",
+		"Principal": {
+			"Service": "transfer.amazonaws.com"
+		},
+		"Action": "sts:AssumeRole"
+		}
+	]
+}
+EOF
+}
+
+resource "aws_iam_policy" "foo" {
+	name 		= "tf-test-transfer-server-iam-policy"
+	path        = "/"
+	description = "IAM policy for for Transfer Server testing"
+  
+	policy = <<EOF
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Effect": "Allow",
+			"Action": "logs:*"
+			"Resource": "*"
+		}
+	]
+}
+EOF
+}
+
+resource "aws_iam_policy_attachment" "foo" {
+	name       = "%s"
+	roles      = ["${aws_iam_role.foo.name}"]
+	policy_arn = "${aws_iam_policy.foo.arn}"
+}
+  
+
+resource "aws_transfer_server" "foo" {
+  identity_provider_type = "SERVICE_MANAGED"
+  logging_role = "${aws_iam_role.foo.arn}"
+
+  tags {
+	NAME   = "tf-acc-test-transfer-server"
+	ENV    = "test"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `invocation_role` - (Optional) The Role parameter provides the type of InvocationRole used to authenticate the user account.
+* `url` - (Optional) - The IdentityProviderDetail parameter contains the location of the service endpoint used to authenticate users.
```

Comment:
This seems to be the AWS SDK documentation for the parameter, which could be improved into more operator friendly documentation, e.g.

```suggestion
* `url` - (Optional) - URL of the service endpoint used to authenticate users with an `identity_provider_type` of `API_GATEWAY`.
```